android/src/main/java/com/bitmovin/player/reactnative/EventRelay.kt

```diff
@@ -0,0 +1,45 @@
+package com.bitmovin.player.reactnative
+
+import com.bitmovin.player.api.event.Event
+import com.bitmovin.player.api.event.EventEmitter
+import kotlin.reflect.KClass
+
+/**
+ * Attaches and detaches listener for the provided forwarding events on the current [EventEmitter] instance and
+ * relays the received events together with their associated name to the provided event output.
+ */
+class EventRelay<E : EventEmitter<T>, T : Event>(
+    /**
+     * List of events that should be relayed and their associated name.
+     */
+    forwardingEventClassesAndNameMapping: Map<KClass<out T>, String>,
+    /**
+     * Is called for every relayed event together with its associated name.
+     */
+    private val eventOutput: (String, Event) -> Unit
+) {
+    private val playerEventListenersMap = mutableMapOf<KClass<out T>, (T) -> Unit>()
```

Comment:
This component is very generic and not coupled to `player` events. Should we rename this property to something more generic as well?

As briefly discussed offline a `List` structure would suffice here.

Reply:
True. Missed that name in the process as it seems.

https://github.com/bitmovin/bitmovin-player-react-native/pull/85/commits/39b28c7e9444dd2e9a0298fdaf3eb9fb04ffd37a